io.gitlab.celleron56.libretile.yaml

```
@@ -0,0 +1,93 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: libretile.sh
+finish-args:
+  - --socket=x11
```

```suggestion
  - --socket=x11
  - --share=ipc
```

this is needed for x11.

---

thx (i thought socket=x11 was enough since i tested it on wayland )

---

This is still missing.